vllm/v1/worker/gpu_model_runner.py

```diff
@@ -726,12 +724,11 @@ def _calc_spec_decode_metadata(
         self,
         scheduler_output: "SchedulerOutput",
         cu_num_tokens: np.ndarray,
-    ) -> Tuple[torch.Tensor, torch.Tensor]:
+    ) -> torch.Tensor:
```

Comment:
This is just fixing incorrect return type.

Reply:
oh thanks for catching this!